fleets/kubevirt/fleet.yaml

```diff
@@ -2,7 +2,7 @@ defaultNamespace: kubevirt
 helm:
   chart: "kubevirt"
   repo: "https://suse-edge.github.io/charts"
-  version: "0.1.0"
+  version: "0.2.4"
   # kubevirt namespace is created by kubevirt as well, we need to take ownership of it, otherwise it fails
```

Comment:
This is no longer true. The namespace creation is up to the user. Can we drop the comment and adjust the fleet to create it?

Reply:
Done